docs/user/bots.rst

```diff
@@ -1243,6 +1243,38 @@ HTML Table Parser
  * `"type"`: set the `classification.type` statically, optional
  * `"html_parser"`: The HTML parser to use, by default "html.parser", can also be e.g. "lxml", have a look at https://www.crummy.com/software/BeautifulSoup/bs4/doc/
 
+JSON Custom Parser
+^^^^^^^^^^^^^^^^^
+
+**Configuration parameters**
+
+* `"json_data_format"`: Boolean, if list of data is within key of json object, optional. Default: false.
```

Comment:
Please expand the explanation, examples are also helpful.

Reply:
json_data_format will be **True** if there is a single json and list of data is inside any of its child nodes. For below json, we find data inside key 'data'
`{
    "complete": false,
    "count": 9632240,
    "data": [
        {
            "ip": "179.124.36.196",
            "classification": "malicious",
            "first_seen": "2020-01-13",
            "last_seen": "2020-12-14",
            "tags": [
                "SSH Scanner",
                "SSH Worm"
            ],
            "cve": []
        },
        {
            "ip": "189.86.227.150",
            "classification": "malicious",
            "first_seen": "2019-01-17",
            "last_seen": "2020-12-14",
            "tags": [
                "Eternalblue",
                "SMB Scanner"
            ],
            "cve": [
                "CVE-2017-0144"
            ]
        }
    ],
    "message": "ok"
}`

Reply:
Can you please add that example to the docs?